frontend/src/Editor/Inspector/EventManager.jsx

```diff
@@ -862,7 +862,7 @@ export const EventManager = ({
     );
   };
 
-  const componentName = componentMeta.name ? componentMeta.name : 'query';
+  const componentName = componentMeta.name ? componentMeta.name : 'page';
```

Comment:
Changing the text will change the default text in the queries as well. Attaching the screenshot

<img width="746" alt="image" src="https://user-images.githubusercontent.com/50441969/230282001-43ed6008-0538-445b-b90a-040bea23c518.png">

Instead of changing the test, pass the `name` key in the `componentMeta` of the pages `settingsModal.jsx`. 

<img width="976" alt="image" src="https://user-images.githubusercontent.com/50441969/230282410-aab04517-5d3e-4061-9940-e9fc8757c041.png">

Reply:
Thanks for the suggestion @kavinvenkatachalam . Updated it